src/applications/ivc-champva/10-10D/components/File/MissingFileOverview.jsx

```diff
@@ -0,0 +1,344 @@
+import React, { useState } from 'react';
+import {
+  VaAlert,
+  VaCheckboxGroup,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { titleUI } from 'platform/forms-system/src/js/web-component-patterns';
+import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
+import PropTypes from 'prop-types';
+import {
+  identifyMissingUploads,
+  getConditionalPages,
+} from '../../helpers/supportingDocsVerification';
+import MissingFileList from './MissingFileList';
+
+export const mailInfo = (
+  <>
+    Mail your files to:
+    <address className="vads-u-border-color--primary vads-u-border-left--4px vads-u-margin-left--3">
+      <p className="vads-u-padding-x--10px vads-u-margin-left--1">
+        VHA Office of Community Care
+        <br />
+        CHAMPVA Eligibility
+        <br />
+        P.O. Box 469028
+        <br />
+        Denver, CO 80246-9028
+        <br />
+        UnitedStates of America
```

Comment:
Does United States need to be 2 words?

Reply:
It definitely should be two words - fixed!